pkg/selinux/selinux.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright © 2025 SUSE LLC
+SPDX-License-Identifier: Apache-2.0
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package selinux
+
+import (
+	"container/ring"
+	"context"
+	"path/filepath"
+
+	"github.com/suse/elemental/v3/pkg/chroot"
+	"github.com/suse/elemental/v3/pkg/sys"
+	"github.com/suse/elemental/v3/pkg/sys/vfs"
+)
+
+const (
+	SelinuxTargetedContextFile = selinuxTargetedPath + "/contexts/files/file_contexts"
+
+	selinuxTargetedPath = "/etc/selinux/targeted"
+)
+
+// Relabel will relabel the system if it finds the context
+func Relabel(ctx context.Context, s *sys.System, rootDir string, extraPaths ...string) error {
+	contextFile := filepath.Join(rootDir, SelinuxTargetedContextFile)
+	contextExists, _ := vfs.Exists(s.FS(), contextFile)
+
+	if contextExists {
+		var err error
+		args := []string{"-i", "-F"}
+
+		// We only keep last 10 lines of the stdout and stderr for debugging purposes
+		stdOut := ring.New(10)
```

Comment:
nit: since 10 lines is something we put also in the following function, it could be nice to have something like a "DEBUGLINES" const at the beginning of the file